components/SideImage.tsx

```diff
@@ -0,0 +1,152 @@
+import Image from "next/image";
```

Comment:
this is the same SideImage component that was in `components/home/` with some edits noted below